assembl/static2/js/app/components/common/termsForm.jsx

```diff
@@ -47,21 +49,25 @@ class DumbTermsForm extends React.Component<TermsFormProps, TermsFormState> {
     }
   };
 
-  handleSubmit = () => {
-    this.props.handleAcceptButton();
+  handleSubmit = (termFormType) => {
+    this.props.handleAcceptButton(termFormType);
     closeModal();
   };
 
   render() {
     const { isScrolled } = this.state;
-    const { isChecked, text, style = {} } = this.props;
+    const { isChecked, termsAndConditionsText, privacyPolicyText, isPrivacyPolicy, style = {} } = this.props;
```

Comment:
I think you should avoid a boolean prop and multiple conditions for this. Here you could have new props: `headerTitle`, `body` (or `text`) and `handleAcceptButton`. It is easier to read and more reusable.